src/api/spec/features/webui/users/notifications_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'browser_helper'
+
+RSpec.describe 'User notifications', type: :feature, js: true, beta: true do
```

Comment:
_Beta_ is set for this whole test, so I'd suggest to move it to `spec/features/beta`. All the tests in that directory has beta enabled by default, so there is no need to add `beta: true`.

That behavior comes from: 
https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/support/beta.rb